### ISSUE TITLE
Add eligibility video link on VALife page

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#56966]
## Issue Description

Steve Fital asked us to add the following YouTube link to our [VALife insurance page](https://www.va.gov/life-insurance/options-eligibility/valife/):

https://www.youtube.com/watch?v=umVFO71Ebos

Before we add the link, we need to confirm that the video is 508 compliant and determine where on this page the link should go.

## Tasks
- [x] Confirm with Steve that video is 508 compliant
- [x] Add link and link text to page
- [x] Review (Laura)
- [x] Copyedit (Megan)
- [x] Publish (Megan)

## Acceptance Criteria
- [x] Video link is live on the site

**Comment by RLHecht** [comment]
Steve said that the comms team told him the contractor made the video compliant. So this should be ready to be pout on the page.

**Comment by RLHecht** [comment]
Steve OK'ed the link and content we added. @megzehn this is ready for you to copyedit and publish!

**Comment by RLHecht** [comment]
Thanks! This is now live. Closing ticket.
